doc/release-notes/release-7.sgml.in

```diff
@@ -123,6 +124,14 @@ in the position of what used to be a %ui record field.
 <p>If necessary, an external ACL helper can be written to perform Ident transactions
 and deliver the user identity to Squid through the **user=** annotation.
 
+<sect1>Removed URN protocol support
+
+<p>Squid URN resolution code has been neglected for a very long time and caused
+multiple security vulnerabilities. This feature was rarely used (if at all).
```

Comment:
Come on be honest. All attempts to update the code were vetoed by you. Otherwise this code would very much have been updated by at least four authors in the past 10 years.

Reply:
> Come on be honest. All attempts to update the code were vetoed by you. Otherwise this code would very much have been updated by at least four authors in the past 10 years.

I hope that any vetoes were correct, but this discussion and implications of dishonesty feel out of scope: PR text describes code state. It does not speculate about the reasons that led to that code state.